pkg/utils/utils.go

```diff
@@ -36,3 +37,15 @@ func IsNonEmptyFile(dir, file string) bool {
 	}
 	return !info.IsDir() && info.Size() > 0
 }
+
+func CreateTempFileWithContent(dir string, dbType string) {
```

Comment:
Consider breaking this out into a separate `pkg/test_utils` library; This function doesn't look like it's intended to be used in production given the baked in assumptions and it's good hygiene to separate out functions with these assumptions.

Also, the latest change seems to have gotten out of sync with dpkg_test.go expectations for the function.

Reply:
good idea! moved